models/banner.js

```diff
@@ -33,12 +33,7 @@ const MODEL = {
         .description('Username of user creating the banner')
         .example('batman123'),
 
-    type: Joi
-        .string().valid([
-            'info',
-            'warn'
-        ])
-        .max(32)
+    type: Joi.string().valid('info', 'warn')
```

Comment:
might still need max length for https://github.com/screwdriver-cd/screwdriver/issues/325?
actually nvm looks like this PR should check: https://github.com/screwdriver-cd/data-schema/pull/280/files
```suggestion
    type: Joi.string().valid('info', 'warn').max(32)
```

Reply:
@tkyi The .max() function is not allowed for .valid(). It has two values that should validate only info and warn.